src/utils/getPackageJson.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import path = require('path');
+import { access } from 'fs';
+import { promisify } from 'util';
+
+const accessAsync = promisify(access);
+
+export async function getPackageJson(appDir: string): Promise<Object> {
+    const packageJsonPath = path.join(appDir, 'package.json');
+    try {
+        await accessAsync(packageJsonPath);
+        return require(packageJsonPath);
```

Comment:
I've always used [`fs-extra`](https://www.npmjs.com/package/fs-extra) for file system operations in my Node.js projects - it just makes a lot of things easier. This repo isn't using it yet, but I planned on adding it eventually. In this case, you would use `fse.readJson` instead of `access` and `require`

Reply:
Thanks! Looks like a pretty helpful library, better than promisifying everything :p